### ISSUE TITLE
Dead links

### ISSUE THREAD

**Issue by HuguesPecout** [magrit-formations/magrit-formations.github.io#1]
Salut,

Les liens vers les différents supports ne fonctionnent plus, y compris le lien vers le site principal : [https://magrit-formations.github.io/](https://magrit-formations.github.io/). Merci d'avance pour la remise en ligne.

Hugues

**Comment by rCarto** [comment]
Est ce que d'autres pages xxx.github.io fonctionnent correctement pour toi depuis la même connexion ?

**Comment by HuguesPecout** [comment]
Oui, par exemple : [https://ee-cist.github.io/TIGR/](https://ee-cist.github.io/TIGR/)
Pquoi, cela fonctionne chez vous ?

**Comment by mthh** [comment]
Oui ça fonctionne avec la connexion que j'utilise actuellement (chez moi), aussi bien magrit-formations que ee-cist (et les autres fois où tu nous avais dit que la page magrit-formations ne fonctionnait pas, on était au bureau et on arrivait bien à accéder... étrange...).

**Comment by HuguesPecout** [comment]
Effectivement c'est très étrange. Je précise qu'il ne fonctionne pas sur mes deux PC (Ubuntu et windows, et mozilla/chrome à chaque fois). Je viens également de les faire tester sur deux autres PC, et cela ne fonctionne que sur un. Est-il possible que cela vienne d'un problème de connexion sécurisé, de cookies ou de proxy ?

**Comment by mthh** [comment]
- Pbm de connexion sécurisée : je ne vois pas vraiment pourquoi, c'est github.io qui gère le certificat (donc c'est le même pour l'ensemble des sites du domaine github.io, donc il n'y a pas de raison que ça créé un pbm de connexion seulement vers certains sites hébergés sur ce domaine)
- De cookies : idem je ne vois pas pourquoi, on n'en utilise pas du côté de magrit-formations.
- De proxy : aucune idée, vous en utilisez un à l'endroit où vous avez des soucis de connexion ?

Par ailleurs, il est écrit quoi dans ton navigateur / dans le navigateur des gens quand ils essaient d'accéder à [https://magrit-formations.github.io/](https://magrit-formations.github.io/) et que ça échoue ? 
Si tu ouvres les outils pour développeur, est-ce qu'il y a un truc intéressant écrit quelque part ? (dans l'onglet "console" ou dans l'onglet "network" par exemple ?)

**Comment by HuguesPecout** [comment]
![image](https://github.com/user-attachments/assets/f5f0f6a7-69b8-4c72-a36b-e18ffc61bea1)

Rien d'intéressant...

**Comment by HuguesPecout** [comment]
![image](https://github.com/user-attachments/assets/5e487435-bf11-4ce6-96b6-2c581e06b42a)

**Comment by mthh** [comment]
OK merci. Effectivement pas grand chose d'intéressant...

Quoi que je constate que si je tape une adresse bidon genre https://sdkfdskfjkdsfjskdfjs.com/ j'ai une erreur `DNS_PROBE_POSSIBLE`.

Alors que là c'est écrit `DNS_PROBE_FINISHED_NXDOMAIN`.

En tapant ce message d'erreur dans un moteur de recherche j'ai vu différents trucs qui parlaient de nettoyer le cache DNS de la machine / du browser. Sur windows tu peux tenter `ipconfig /flushdns` dans un terminal.

Sinon ça vaut peut-être le coup de demander à la DSI locale, de leur montrer qu'en 4G vous pouvez y accéder mais que sur leur réseau vous ne pouvez pas y accéder... (si jamais cette URL avait été bloqué par exemple ? car je pense que si c'était le cas ça mettrait peut-être bien ce message `DNS_PROBE_FINISHED_NXDOMAIN`...).

Et / ou sinon ils auront peut-être une meilleure idée que moi.

@rCarto une idée ?

**Comment by rCarto** [comment]
Je crois que j'avais eu le même soucis à l'INED, soit le bâtiment d'à côté dans le campus. 


> Effectivement c'est très étrange. Je précise qu'il ne fonctionne pas sur mes deux PC (Ubuntu et windows, et mozilla/chrome à chaque fois). Je viens également de les faire tester sur deux autres PC, et cela ne fonctionne que sur un. Est-il possible que cela vienne d'un problème de connexion sécurisé, de cookies ou de proxy ?

toujours sur le campus on est d'accord?


Aucune idée du pourquoi cependant.

**Comment by rCarto** [comment]
Il nous manque une information @HuguesPecout . Quelle connexion utilises tu ? Wifi avec Eduroam, sans Eduroam? Filaire dans ton bureau à Condorcet? L'hypothèse est que le site soit bloqué au niveau de Condorcet.

**Comment by mthh** [comment]
@rCarto j'ai eu la confirmation que ça ne fonctionne pas dans les locaux du campus Condorcet :
- ni en filaire
- ni sur le WiFi eduroam
- ni sur le WiFi "invité"

**Comment by mthh** [comment]
Salut @HuguesPecout !
À tout hasard, pourrais-tu tester sur le campus Condorcet et me dire si le problème est toujours présent ? Merci !

**Comment by mthh** [comment]
Visiblement ça ne fonctionne toujours pas, on va envisager une migration vers un autre nom de domaine...

**Comment by mthh** [comment]
Fait, et maintenant ça fonctionne!
